src/assets/TokenRatesController.ts

```diff
@@ -258,6 +273,22 @@ export class TokenRatesController extends BaseController<
     }
   }
 
+  /**
+   * Fetches supported vsCurrencies from CoinGecko API
+   *
+   * @returns Array of supported vsCurrencies by CoinGecko API
+   */
+  async fetchSupportedVsCurrencies(): Promise<string[]> {
+    try {
+      const supportedVSCurrencies: string[] = await handleFetch(
+        CoinGeckoApi.getSupportedVsCurrencies(),
+      );
+      return supportedVSCurrencies;
+    } catch {
+      return [];
```

Comment:
It looks like errors are still being suppressed here, and below in `checkIsSupportedVsCurrency`

Reply:
Removing the try/catches here!

Reply:
@wachunei same thing here @Gudahtt's point that errors are getting swallowed applies to [`fetchSupportedChains`](https://github.com/MetaMask/controllers/pull/585/files/266f68fd9540b0c08953cbfbf1eeade8697c05ed#diff-957c27782cf52be9a410d31f752cc1ab5fea70706f6ee092f7af158e6449d9a3L251). Any objection to my removing the try/catch? Do you know if there are any errors that we want to handle particularly?

Reply:
I think this might only raise network errors, I'm okay removing the catch blocks if we are certain these will be handled

Reply:
Yes, ultimately this gets wrapped in `safelyExecute` which logs the error out. @Gudahtt has [some feelings](https://github.com/MetaMask/controllers/pull/585#discussion_r710255154) about this approach but for now this is what we're doing

Reply:
I will remove the try/catch from fetchSupportedChains